devices/hive.js

```diff
@@ -144,6 +145,9 @@ module.exports = [
                 .withDescription('Whether or not the unit needs warm water. `false` No Heat Request or `true` Heat Request'),
             exposes.climate().withSetpoint('occupied_heating_setpoint', 5, 32, 0.5).withLocalTemperature().withPiHeatingDemand()
                 .withSystemMode(['heat']),
+            exposes.numeric('external_measured_room_sensor', ea.ALL)
+                .withDescription('Set at maximum 3 hours interval but not more often than every 30 minutes at every 100 ' +
+                    'value change. Resets every 3hours to standard (-8000=undefined).'),
```

Comment:
Is it reportable? If so maybe it will report when it is reset after 3 hours, that way the field can be automatically updated.

Reply:
@sjorge The attribute is defined as Fixed reporting. 
![image](https://user-images.githubusercontent.com/9019306/128836623-6fd2ae71-9f15-4bca-a0cf-4e66e15d2cb4.png)

The field is storing a value that you pass to it for temperature coming from an external sensor. There is not "time left" attribute reported back.

I think the way this is designed to work is for people that have inaccurate temp reading from the radiator compared to the rest of the room. Case in which they will be sending values constantly, since their problem will persist? The temp can only be set via gateway (z2m) so updating it will keep the latest value in z2m, and every change to the value will always be set by commands.

I do not know how beneficial reporting will be? I mean i guess it will be if someone does it once, and then they stop. But I guess people will either use it or not, with no in between. 

Here is what I added:
![image](https://user-images.githubusercontent.com/9019306/128838109-a4ce466c-4970-47aa-995c-ba81a1f1c0c0.png)

Reply:
I guess it depends, if you set it now and read it back in 3h 10m ... does it still return the value that was set? Or does it return -8000, if the later it would make sense to have it report the value. If reading the value after the time returns the set value, there is no need.

Reply:
It would automatically change to -8000 (that is according to the specs document, TBH i have not tried it to see what happens) - and it does make sense for this behaviour otherwise the internal PID would never know to go back to standard operation.

Well I have done the commit for reporting, the reporting is set successfully so the device seems to accept the config.

For testing, with reporting configured, I have just set a value. I will report back in 3 hours.

Reply:
@sjorge Reporting works as expected. Value changed from my set one of 2400 to -8000 when the 3 hours have elapsed. That said, i will mark this thread as resolved and I guess the PR is good for merging.

Reply:
Excellent!